pkg/application/options.go

```diff
@@ -184,19 +144,20 @@ func WithConfigFlags(args []string, flags ...func() (string, string, any, string
 
 func WithConfigFileFlag(app *App) {
 	app.addConfigOption(func(config cfg.GosoConf) error {
-		flags := flag.NewFlagSet("cfg", flag.ContinueOnError)
-		configFile := flags.String("config", "", "path to a config file")
+		var opts struct {
+			Config []string `long:"config" short:"c"`
+		}
 
-		args := funk.Filter(os.Args[1:], func(arg string) bool {
-			return strings.Contains(arg, "config")
-		})
+		if _, err := flags.NewParser(&opts, flags.IgnoreUnknown).ParseArgs(os.Args[1:]); err != nil {
```

Comment:
I guess it is okay to ignore the first arg, but if it is supported, I guess it would be simpler to just pass all args

Reply:
true